### PR TITLE
Fixed #7 and a bug where a disabled switch would still move

### LINKED ISSUE #7: Weird behaviour when you cancel the gesture
If you try to swipe the switch in one way and then keep the finger pressed and swipe the other way, it doesn't cancel the state of the switch which stays in middle of animation

### PR DESCRIPTION
**Pull request by VinceBT**
+ Fixed handler width bug
+ Fixed disabled switch behaviour
+ Updated example
+ Example now works also for Android
+ Updated .npmignore and added .gitignore for future collaborations

### CODE REVIEW

**Comment by poberwong** [comment]
I'm too busy to review the pull request recently. I will review it as soon as I have time

**Comment by tbazko** [comment]
Would be great to have this fix!

**Comment by poberwong** [comment]
You'd better do something with more commits instead of only one.  
"Fixed disabled switch behaviour and update Example" has already been fixed and you can create new pull request to fixed following: 

* Fixed disabled switch behaviour
* Updated .npmignore and added .gitignore for future collaborations
* Example now works also for Android

It could be better to use single commit to each change in lib/index and it could be easier for me to add tags and publish new version on npm. 

In addition，remember to change version code in package.json if necessary

**Comment by VinceBT** [comment]
I'll do that for future pull requests.
I don't have the time to do what you're asking for and you should be the one to manage versions btw, it's your package
I've resolved the conflitcs generated by your fixes.

File: lib/index.js [hunk 1]
```diff
@@ -141,7 +135,7 @@ export default class extends Component {
     Animated.timing(handlerAnimation,
       {
         toValue: value,
-        duration: 200,
+        duration: 50,
```
Comment (poberwong) [inline lib/index.js]: I don't think it is necessary to change duration into 50ms.

**Pull request closed without merging**